cfme/common/__init__.py

```diff
@@ -293,8 +293,8 @@ def get_tags(self, tenant="My Company Tags"):
             List of tags in format "Tag_category: Tag_name"
         """
         view = navigate_to(self, 'Details')
-        # look for simple 'tag' widget first, then standard entities.smart_management
-        tag_table = getattr(view, 'tag', view.entities.smart_management)
+        # look for simple 'smart_management' widget first, then standard entities.smart_management
+        tag_table = getattr(view, 'smart_management', view.entities.smart_management)
```

Comment:
**Optional**
Add TODO here to remove this double check, we should have consistent DetailsView.entities.smart_management implementation across framework. Might be able to remove now if you check for DetailsViews without nested entities.

Reply:
Added TODO here, as we still have places with smart_management widget in view.